release notes/v0.37.0.md

```diff
@@ -0,0 +1,72 @@
+
+It's important to highlight that an old API is deprecated and will be removed in the next k6's release (`v0.38.0`). Every extension that isn't implementing the API will get the warning message, like:
+
+```sh
+WARN[0000] Module 'k6/x/sql' is using deprecated APIs that will be removed in k6 v0.38.0, for more details on how to update it see https://k6.io/docs/extensions/guides/create-an-extension/#advanced-javascript-extension
+```
+
+We did migrations for some xk6 extensions (see connected issues to the task [#2344](https://github.com/grafana/k6/issues/2344)). The pull requests can serve as examples.
+
+## Enhancements and UX improvements
+
+- We continued work on the source map feature: add samples ([#2339](https://github.com/grafana/k6/pull/2339)) and improve loading and parsing ([#2355](https://github.com/grafana/k6/pull/2355)).
+- Updated installation from source instructions ([#2359](https://github.com/grafana/k6/pull/2359)). Thanks, @[karitham](https://github.com/Karitham), for making the change.
+- We migrated our docker repository to the [grafana/k6](https://hub.docker.com/r/grafana/k6) ([#2377](https://github.com/grafana/k6/pull/2377)).
+- Introduced a Go-based thresholds parsing ([#2400](https://github.com/grafana/k6/pull/2400)).
+- Enabled more TC39 tests: [optional-chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) ([#2338](https://github.com/grafana/k6/pull/2338)) and async ([#2396](https://github.com/grafana/k6/pull/2396)).
+
+## Bugs fixed!
+
+- Prevent panic in [check](https://k6.io/docs/javascript-api/k6/check-val-sets-tags) if a second argument isn't provided ([#2387](https://github.com/grafana/k6/pull/2387)).
+- `http.head` stopped taking a body as a second argument ([#2402](https://github.com/grafana/k6/pull/2402)). A signature looks like how it was before v0.36.0 `http.head(url, [params])`.
+- Fixed [options.scenarios'](https://k6.io/docs/using-k6/options/#scenarios) JSON marshaling ([#2392](https://github.com/grafana/k6/pull/2392)).
+- Prevent panic in [http.batch](https://k6.io/docs/javascript-api/k6-http/batch-requests) if pass no arguments (https://github.com/grafana/k6/pull/2415).
+
+## Internals
+
+- We updated our CI to make a developer experience better. Dependency and linter checks run only for pull requests ([#2403](https://github.com/grafana/k6/pull/2403)). We also migrated to a native `golangci-lint` GitHub action. Also bumped the windows version ([#2388](https://github.com/grafana/k6/pull/2388)).
+- Updated goja dependency ([#2350](https://github.com/grafana/k6/pull/2350)).
```

Comment:
no certain we need to mention this - we usually do as it adds something but there has been nothing new at this point and the minor fixes are irrelevant to (probably) all users.